src/main/kotlin/next/step/racing/domain/Cars.kt

```diff
@@ -0,0 +1,27 @@
+package next.step.racing.domain
+
+data class Cars(val cars: List<Car>) {
+    companion object {
+        fun init(count: Int): Cars {
+            return Cars(IntRange(1, count).map { Car.init() })
+        }
+
+        fun at(count: Int, pos: Position): Cars {
+            return Cars(IntRange(1, count).map { Car.at(pos) })
+        }
+    }
+
+    fun move(drivingStrategy: () -> Int): Cars {
+        return Cars(
+            cars.map { car ->
+                car.move(drivingStrategy)
+            }
+        )
+    }
+
+    fun positions(): List<Int> {
+        return cars.map { car ->
+            car.position()
+        }
+    }
```

Comment:
중요한 부분은 아닌데, 이렇게 체이닝으로만 끝나는 경우 return 구문을 생략하고 아래처럼 사용하실 수 있어요.

```suggestion
    fun positions(): List<Int> = cars.map { car -> car.position() }
```